package.json

```diff
@@ -3,7 +3,6 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "bootstrap": "^4.3.1",
```

Comment:
This comment is unrelated to this PR, more of a general one for the project:

My preference for dependencies declared in `package.json` is to lock down your versions. For example, this line could be:

- `"bootstrap": "4.3.1",` --> only `4.3.1`
- `"bootstrap": "~4.3.1",` --> `4.3.1` or `4.3.2`, `4.3.3`, etc (if more patch versions get released)

Right now, if you run the installation, you could get `4.3.1`, `4.4.0`, or `4.4.1` ... basically anything until `5.x.x` gets released.

Reply:
... and bonus: Much less diff-noise in `package-lock.json` and `yarn.lock` files.